android/src/main/kotlin/com/game/quillyjumper/android/Launcher.kt

```diff
@@ -1,13 +1,13 @@
-package com.game.quillyjumper
+package com.game.quillyjumper.android
 
 import android.os.Bundle
 import com.badlogic.gdx.backends.android.AndroidApplication
 import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration
+import com.game.quillyjumper.Main
 
-class AndroidLauncher : AndroidApplication() {
+class Launcher : AndroidApplication() {
```

Comment:
keep the name `AndroidLauncher` because it makes it clearer when searching `Launcher` classes. There is a `DesktopLauncher` for desktop and an `AndroidLauncher` for android.